cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -2205,6 +2205,29 @@ ntel/x86_64/2013/composer_xe_2013/composer_xe_2013_sp1.3.174/mkl/include </base>
   </SLIBS>
 </compiler>
 
+<compiler MACH="jlse-iris" COMPILER="intelGPU">
```

Comment:
Very minor issue: can we rename to intelgpu to be consistent with similar naming scheme for Summit etc.?

Reply:
Yes, we can do `intelgpu` instead.